backend/src/main/validation/PromotionValidation.ts

```diff
@@ -20,17 +31,30 @@ export class PromotionValidation {
       .required(),
     name: Joi.string().required(),
     description: Joi.string().required(),
-    expirationDate: Joi.date().required(), // note incoming format is a string, and Joi will automatically convert to type Date
+    /**
+     * startDate is NOT required. If is null/undefined, then default to current date.
+     * However, Promotion.startDate cannot be null or undefined
+     * * note incoming format is a string, and Joi will automatically convert to type Date
+     * * startDate cannot be larger than expirationDate
+     * */
+    startDate: Joi.date()
+      .max(Joi.ref('expirationDate'))
+      .empty(null)
+      .default(new Date()),
+    /** note incoming format is a string, and Joi will automatically convert to type Date */
```

Comment:
@andrewzulaybar just moving what we talked about here so everyone can input ideas.

Right now we are thinking when the user is choosing the start date of a promotion, if they indicate somehow that the promotion is active (e.g. ticking a checkbox rather than selecting a start date), then FE will leave startDate as null/undefined when they send the POST request to promotions. 

This will set the start date of the promotion automatically to the current date.

The reasoning for this is because say we want to show in the promotions card whether a promotion is active or whether it has started or not. We can then use date comparison tools, e.g. if `promotion.startDate <= <today's date>` then we will show that the promotion is active, otherwise we would show the promotions start date.